docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -0,0 +1,105 @@
+= Kubernetes Operator (Preview)
```

Comment:
Since this is a tutorial, a better title would be 'Deploy a Cluster with the Enterprise Operator'

Reply:
Changed.